trp_project_scrum/README.rst

```diff
@@ -0,0 +1,100 @@
+====================================
```

Comment:
You can not use the oca-gen-addon-readme generator, because this is not an OCA module. Or actually you might, if you use the very latest version of that tool and you use the --org-name argument.